sys/netbsd/sys.txt

```diff
@@ -190,6 +194,46 @@ ucred {
 	gid	gid
 }
 
+posix_spawn_file_actions {
+	size	int32
+	len	int32
+	fae	ptr[in, posix_spawn_file_actions_entry]
+}
+
+posix_spawn_file_actions_entry {
+	fae_action	int32
+	fae_filedes	int32
```

Comment:
Based on the name and fae_action values, this looks like a file descriptor rather then an integer.